Deploy/terraform/modules/iam/main.tf

```diff
@@ -280,10 +455,16 @@ resource "aws_iam_role_policy_attachment" "instance_role_cloud_watch_logs_policy
   policy_arn = "${aws_iam_policy.cloud_watch_logs_policy.arn}"
 }
 
-resource "aws_iam_role_policy_attachment" "instance_role_bfd_opt_out_policy_attach" {
-  role       = "${aws_iam_role.ab2d_instance_role.name}"
-  policy_arn = "${aws_iam_policy.bfd_opt_out_policy.arn}"
-}
+#######################################
+#
+# END #2
+#
+#######################################
+
+# resource "aws_iam_role_policy_attachment" "instance_role_bfd_opt_out_policy_attach" {
+#   role       = "${aws_iam_role.ab2d_instance_role.name}"
+#   policy_arn = "${aws_iam_policy.bfd_opt_out_policy.arn}"
+# }
```

Comment:
Again all of these comments in terraform can we just delete this code?

Reply:
Cleaned up in later pull request.